neuror/cli.py

```diff
@@ -349,16 +350,15 @@ def folder(input_dir, output_dir, width, display, plane):
     '''Compute cut planes for all morphology in INPUT_DIR and save them into OUTPUT_DIR
 
     See "cut-plane compute --help" for more information'''
-    for f in iter_morphology_files(input_dir):
-        L.info('Seaching cut plane for file: %s', f)
-        inputfilename = os.path.join(input_dir, f)
-        outfilename = os.path.join(output_dir, os.path.basename(f) + '.json')
+    for inputfilename in iter_morphology_files(input_dir):
+        L.info('Seaching cut plane for file: %s', inputfilename)
+        outfilename = os.path.join(output_dir, inputilename.name + '.json')
```

Comment:
```suggestion
        outfilename = os.path.join(output_dir, inputilename.with_suffix('.json'))
```

Reply:
That makes a difference in the outcome. 
first one is 'a.h5.json' second one is 'a.json'
I think 2nd one is what we want though